README.md

```diff
@@ -0,0 +1,54 @@
+# Event Log Uploader
+NOTICE: _Use **Python 3.10** to avoid issues with dependencies._
+
+## Install dependencies
+```bash
+pip install -r requirements.txt
+```
+
+## Run
+NOTICE: _All command line params except `-config` can be set in config file._
+
+* upload data 
+  ```sh
+  pfutil  -c 'path to congig yaml file' put -e 'path to eventlog csv file' --csv-out 'relative path to output directory' 
+  ```
+
+* display help
+  * `pfutil -h` prints available commands and top level options,
+  * `pfutil <COMMAND> -h` shows command's help, ex: `pfutil put -h`
+  
+### Example
+
+[example_data/p2p/](example_data/p2p/) directory contains sample data
+```sh
+./pfutil -c example_data/p2p/config.yaml  put -e example_data/p2p/eventlog.csv --csv-out output
```

Comment:
Please add executable flag  (`chmod +x ./pfutil`) to `./pfutil` file to be able to run this command